src/styles/main.scss

```diff
@@ -1,3 +1,16 @@
-body {
-  margin: 0;
+@import "../styles/utils/variables";
+@import "utils/reset";
+@import "utils/normalize";
+
+@import "blocks/logo.scss";
+@import "blocks/header.scss";
+@import "blocks/nav.scss";
+@import "blocks/catalog.scss";
+@import "blocks/card.scss";
+@import "blocks/stars.scss";
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
 }
```

Comment:
Maybe better remove this to normalize.scss?)

Reply:
Agree, move this to normalize